Cleanup - Updates

Sicprofundus opened the pull request:
- changed return type of LuaScriptStatus
- removed the unnecessary case-insensitive check (the return will always be in caps)

Sicprofundus committed:
Cleanup - Updates

- changed return type of LuaScriptStatus
- removed the unnecessary case-insensitive check (the return will always be in caps)

brainiac commented on MQ2Status.cpp:
```diff
@@ -64,12 +64,15 @@ std::string to_string_with_precision(const T a_value, const int n = 6)
 		sprintf_s(luaScript, "${Lua.Script[%s].Status}", scriptname);
 		ParseMacroData(luaScript, 64);
+		std::ostringstream out;
+		outputBuffer << luaScript;
```
you do not need to use an ostringstream at all. You can convert to string directly without it.

Sicprofundus committed:
Cleanup

- adjust ostringstream to std::string
- I was hoping I could use fmt::format and only use outputBuffer, instead of 2 variables but when i go to try and parse the data, it gets sideways

Sicprofundus committed:
Variable Cleanup

- moved variable in-scope of usage
- adjusted initialization of variable
